client/lib/analytics/index.js

```diff
@@ -400,13 +403,26 @@ const analytics = {
 		} );
 
 		// Google Analytics
-		const flags = [
+		let flags = [
 			isNewUser && 'is_new_user',
 			isNewSite && 'is_new_site',
 			hasCartItems && 'has_cart_items',
 		].filter( flag => false !== flag );
+
 		analytics.ga.recordEvent( 'Signup', 'calypso_signup_complete:' + flags.join( ',' ) );
 
+		if ( isNewishUserSite ) {
+			// Tracks
+			analytics.tracks.recordEvent( 'calypso_new_user_site_creation', {
+				flow: flow,
+				has_cart_items: hasCartItems,
+			} );
+
+			// Google Analytics
+			flags = [ hasCartItems && 'has_cart_items' ].filter( flag => false !== flag );
+			analytics.ga.recordEvent( 'Signup', 'calypso_new_user_site_creation:' + flags.join( ',' ) );
```

Comment:
I know we discussed this prior, but now that I'm seeing it in code I think it would be OK to omit the `has_cart_items` flag from the GA event, because it would make it less error prone when used in Google Optimize; i.e., others don't need to use "starts with" as the matching technique. If there are never any flags, we can just use the event action "equals" `calypso_new_user_site_creation` to check for this.

I would also remove it from the Tracks event `calypso_new_user_site_creation`, because it's already included in `calypso_signup_complete` above. What do you think?